README.md

```diff
@@ -80,7 +80,7 @@ gem install arkana
 
 # Usage
 
-Arkana requires the declaration of a YAML config file, named e.g. `.arkana.yml`. See [template.yml](/template.yml) for practical examples.
+Arkana requires the declaration of a YAML config file. Although you can name it whatever, the convention is to name it `.arkana.yml`. See [template.yml](/template.yml) for practical examples.
```

Comment:
Sorry for the unrelated change 😅 but it's something I'd like to push forward as well